internal/pkg/service/appsproxy/proxy/proxy_test.go

```diff
@@ -119,6 +126,94 @@ func TestAppProxyRouter(t *testing.T) {
 			},
 			expectedNotifications: map[string]int{},
 			expectedWakeUps:       map[string]int{},
+			expectedSpans: []tracetest.SpanStub{
+				{
+					Name:     "keboola.go.common.dependencies.NewBaseScope",
+					SpanKind: 1,
```

Comment:
There are options to filter out unwanted spans in tests, do we need all these spans in the asset?

Reply:
I would rather check all spans in order to know `name` etc. No more than 1 test for this case is needed. It is always possible to extend it with filter in future.

Reply:
ok